src/utils/IdealGraphVisualizer/Bytecodes/src/main/java/com/sun/hotspot/igv/bytecodes/BytecodeViewTopComponent.java

```diff
@@ -169,23 +170,21 @@ protected boolean requestFocusInWindow(boolean temporary) {
     }
 
     @Override
-    public void resultChanged(LookupEvent lookupEvent) {
-        final InputGraphProvider p = LookupHistory.getLast(InputGraphProvider.class);//)Utilities.actionsGlobalContext().lookup(InputGraphProvider.class);
-            SwingUtilities.invokeLater(new Runnable() {
-                @Override
-                public void run() {
-                if (p != null) {
-                    InputGraph graph = p.getGraph();
+    public void resultChanged(LookupEvent e) {
+        SwingUtilities.invokeLater(new Runnable() {
+            public void run() {
+                final InputGraphProvider provider = LookupHistory.getLast(InputGraphProvider.class);
```

Comment:
Suggestion: use a multi-line lambda for conciseness, like so:
```
        SwingUtilities.invokeLater(() -> {
                final InputGraphProvider provider = LookupHistory.getLast(InputGraphProvider.class);
```

Reply:
done